### PR TITLE
Add events to make calculator work

### PR DESCRIPTION
**Pull request by disc3110**
Added Calculator components
Added Event handlers for buttons.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,31 +1,51 @@
...
+      },
+    };
+  }
+
+  handleEvaluation(button) {
+    const { calcObj } = this.state;
+    const { total, next, operation } = calculate(calcObj, button);
+    this.setState({ calcObj: { total, next, operation } });
+  }
+
```
Comment (Dipeshtwis) [inline src/components/Calculator.js:24]: - [ ] The calculator is working fine but Whenever I'm clicking on any operation, it shows `0` in the display. Please fix this issue. Ex :


 - If I try to do 4 + 5 , then I won't able to see the + sign on the screen. Instead I'm seeing `0` there which confuses the user sometimes. I invite you to change it up a bit (in terms of the display), a user to see the digits (and, hopefully, operations that get typed for great User experience.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,44 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Movies revies Captones
+> This page is a calculator where you can do all basic operations
```
Comment (Dipeshtwis) [inline README.md]: - [ ] I think the existing title doesn't make sense here, it should be something related to the project like `Calculator App`. Please fix it.

File: src/components/Calculator.js [hunk 3]
```diff
@@ -1,31 +1,51 @@
...
+        <button className="lightgray" type="button" onClick={() => this.handleEvaluation(9)}>9</button>
+        <button className="orange" type="button" onClick={() => this.handleEvaluation('x')}>x</button>
+        <button className="lightgray" type="button" onClick={() => this.handleEvaluation(4)}>4</button>
+        <button className="lightgray" type="button" onClick={() => this.handleEvaluation(5)}>5</button>
+        <button className="lightgray" type="button" onClick={() => this.handleEvaluation(6)}>6</button>
+        <button className="orange" type="button" onClick={() => this.handleEvaluation('-')}>-</button>
+        <button className="lightgray" type="button" onClick={() => this.handleEvaluation(1)}>1</button>
+        <button className="lightgray" type="button" onClick={() => this.handleEvaluation(2)}>2</button>
+        <button className="lightgray" type="button" onClick={() => this.handleEvaluation(3)}>3</button>
+        <button className="orange" type="button" onClick={() => this.handleEvaluation('÷')}>÷</button>
```
Comment (Dipeshtwis) [inline src/components/Calculator.js]: [OPTIONAL]

- In general, the calculator has an `add` button just above the equal button and is also given in the design. The symbols are interchanged here for `+` and `÷`. I would invite you to change for a better UI experience.

### Your design

Screenshot from 2021-08-13 10-23-23

### Given design 

Screenshot from 2021-08-10 20-48-42

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @disc3110, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Linters are passing :white_check_mark: 
:+1: You have implemented the click event for all buttons.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**_Check the comments under the review_**

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (DeliceLydia) [reply src/components/Calculator.js:24]: Well addressed this perfectly

File: README.md (hunk 2 above)
Comment (DeliceLydia) [reply README.md]: Well resolved this!

File: src/components/Calculator.js (hunk 3 above)
Comment (DeliceLydia) [reply src/components/Calculator.js]: Well addressed this!

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉 🎉 

### Hi @disc3110 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**